datetime/datetime.go

```diff
@@ -192,11 +193,32 @@ func (dtime *Datetime) add(ival Interval, positive bool) (*Datetime, error) {
 	newVal.Sec += direction * ival.Sec
 	newVal.Nsec += direction * ival.Nsec
 
+	newLoc := dtime.time.Location()
```

Comment:
What's the difference between location and zone?

Reply:
"Europe/Moscow" and "MSK".